Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chromium/ChromiumSecurityWarningsRegistryCleanUpStrategyFactory.cs

```diff
@@ -46,14 +52,15 @@ public void CleanUp ()
       }
     }
 
-    private const string c_chromePoliciesPath = @"SOFTWARE\Policies\Google\Chrome";
     private const string c_securityWarningsPolicyName = "CommandLineFlagSecurityWarningsEnabled";
 
     private readonly ChromiumDisableSecurityWarningsBehavior _disableSecurityWarningsBehavior;
+    private readonly string _policiesPath;
 
-    public ChromeSecurityWarningsRegistryCleanUpStrategyFactory (ChromiumDisableSecurityWarningsBehavior disableSecurityWarningsBehavior)
+    public ChromiumSecurityWarningsRegistryCleanUpStrategyFactory (ChromiumDisableSecurityWarningsBehavior disableSecurityWarningsBehavior, string policiesPath)
```

Comment:
this should not be passed via ctor. the regpath is integral part of this class, not something to be configured on the callsite.

Reply:
Updated in https://github.com/re-motion/Trunk-SVN2Git/pull/17/commits/399f845ca30dd829374a22192d941e36c850603b. Is it better that way?

Reply:
Yes, #399f845 works. please use CreateForChrome and CreateForEdge as factory method names.